infrastructure/helm-chart/charts/components/charts/api/charts/api-communication/templates/communication/deployment.yaml

```diff
@@ -22,7 +22,7 @@ spec:
     metadata:
       labels:
         app: api-communication
-        WorkerType: fargate
+        WorkerType: "{{ default "NodeGroup" .Values.global.workerType }}"
```

Comment:
If one wants to deploy to `Fargate`, he should set `.Values.global.workerType=fargate` correct?

Reply:
yes